batchflow/once_pipeline.py

```diff
@@ -164,6 +193,12 @@ def load_model(self, mode, model_class=None, name=None, *args, **kwargs):
     def _exec_load_model(self, action):
         self.pipeline._exec_load_model(None, action)        # pylint:disable=protected-access
 
+    def import_model(self, model, pipeline=None, name=None):
+        """ Import a model from another pipeline"""
+        return self._add_action(IMPORT_MODEL_ID, _args=dict(source=model, pipeline=pipeline, model_name=name))
+
+    def _exec_import_model(self, action):
+        self.pipeline._exec_import_model(None, action)      # pylint:disable=protected-access
```

Comment:
All the import functionality should be moved here (like init_model and init_variable) and Pipeline.import_model should call before.import_model (like init_model)